### ISSUE TITLE
Web Remote/Cast/Audio Settings/Plug-Ins all need adjusting for light mode

### ISSUE THREAD

**Issue by KillerParticles** [yazninja/apple-cider-lite#6]
TLDR is they are in dark mode and not light mode when the theme is in light mode.
I checked the other menus under the drop down menu and they are fine so its just these 4.

![image](https://user-images.githubusercontent.com/93286201/159275655-5a22cce7-a1db-46fb-a2fc-32c3c4ddb6de.png)
Web remotes not too bad, but the text is a bit faded.

![image](https://user-images.githubusercontent.com/93286201/159275777-cabb2bf9-3819-43f4-abd1-d9c6aa538cb8.png)
Cast is unreadable lol, defiantly needs a colour change.

![image](https://user-images.githubusercontent.com/93286201/159275948-868cd7bd-ccb3-4e2e-9870-4c51285d0f12.png)
![image](https://user-images.githubusercontent.com/93286201/159276159-12a30d38-a01a-4d0b-8e72-595f10dd4855.png)
Audio settings and Plug-Ins isn't too bad, but is in dark mode when the theme is in light mode.
